module/module.go

```diff
@@ -413,17 +413,23 @@ func (m *Module) Ready(ctx context.Context, req *pb.ReadyRequest) (*pb.ReadyResp
 	m.mu.Lock()
 	defer m.mu.Unlock()
 	m.parentAddr = req.GetParentAddress()
-	if err := m.connectParent(ctx); err != nil {
-		// Return error back to parent if we cannot make a connection from module
-		// -> parent. Something is wrong in that case and the module should not be
-		// operational.
-		return nil, err
+	if os.Getenv("VALID_PARENT_SOCK") != "false" {
```

Comment:
Can we leave a comment above this explaining the function of this environment variable?

Reply:
Agreed. It might also be helpful to have a variable name that:

1) is namespaced to viam
2) describes the intent more clearly

e.g. `VIAM_NO_MODULE_PARENT` or something like that.